package.json

```diff
@@ -1,6 +1,7 @@
 {
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/235121493-67c879ff-bdfc-48ae-be41-f4c075d2520f.png)

**[OPTIONAL]**

- [ ] It would be nice if you make your commit messages more meaningful and capitalized 👍 because some of your commit messages are not capitalized 😨 Please consider this issue to make your pull request professional and descriptive ☺️